### ISSUE TITLE
[Go] Ability to remove class prefix from methods?

### ISSUE THREAD

**Issue by hazcod** [swagger-api/swagger-codegen#11978]
##### Description

When generating Go code for swagger openAPI 3.0, the Go methods contain the class names as a prefix.
Is there a way to remove those? e.g.

```go
func (a *ProgramUpdatesApiService) ProgramUpdatesDelete
```
vs
```go
func (a *ProgramUpdatesApiService) UpdatesDelete
```

##### Swagger-codegen version

3.0.36

##### Command line used for generation

```shell
swagger-codegen generate -i foo.com/swagger.json -l go -o subdirectory/ --model-package model --api-package api --invoker-package generated
```

**Comment by enricorotundo** [comment]
Got the same question when generating Python code. It seems that the prefix come from the `@ID` annotation.
For instance, I've changed my `// @ID                   pkg/apiServer.submit` to `// @ID                   submit` and the related python method is now called `submit` instead of the `pkgapiserver_submit`. 
Not sure if this is a standard practice, but hope that helps.

**Comment by hazcod** [comment]
@enricorotundo I'm not sure if that's a good solution since an ID should be unique across your namespace.

**Comment by enricorotundo** [comment]
Agreed. I don't have ID collisions in my project but yeah there should be a supported way to handle this properly
